--- shared/ToolchainGCC.cmake.orig	2025-06-05 06:39:20.229266680 +0000
+++ shared/ToolchainGCC.cmake	2025-06-05 06:39:31.699266275 +0000
@@ -34,7 +34,7 @@
 set(CMAKE_MAKE_PROGRAMM "${CMAKE_SYSTEM_LIBRARY_PATH}/bin/make")
 
 # specify to use secure settings 
-set(XMP_ENABLE_SECURE_SETTINGS "ON")
+set(XMP_ENABLE_SECURE_SETTINGS "OFF")
 
 # search for programs in the build host directories
 set(CMAKE_FIND_ROOT_PATH_MODE_PROGRAM NEVER)
